File: Python/Ejercicio 3/main.py
acumulador=0
contador =0
nua = 1

while (nua != 0):
    nua = int(input("ingrese un numero: "))
    acumulador = acumulador + nua
    contador = contador + 1
    print(acumulador, contador)

contador= contador - 1
promedio = acumulador/contador
print("el promedio es: ", promedio)

File: C++/Ejercicio 3/main.cpp
#include <iostream>

using namespace std;

int main()
{
    int numeroA, acumulador = 0, contador =0;
    double promedio;

    do
{
cout << " ingrese un numero " << endl;
cin >> numeroA;


///acumulador = acumulador + numeroA;
acumulador += numeroA ;
contador++;

}while (numeroA != 0);


promedio = double (acumulador)/contador;
        //cast estatico
cout << "promedio" << promedio;

    return 0;
}
